### ISSUE TITLE
[Blog and eBook] Email unsubscribe link does not work

### ISSUE THREAD

**Issue by paagla123** [coa0329/BhaangWeb#120]
This one is URGENT! Needs to be handled in Staging today.

Please href to Studio's default unsubscribe link 
href="##Unsubscribe RedirectURL="http://studio.m.mdl.io/unsubscribe" ##"
Unless a link is already provided in the settings section Unsubscribe field, in which case use the link from settings. We need to make sure the link works.

If in settings only the page name (e.g. remove.html) is provided with no full URL, then please use the domain for the campaign followed by "/pagename.html" in the href. So the complete code will look like: href="##Unsubscribe RedirectURL="http://dvnnnnn.m.mdl.io/<campaignID>/<pagename>.html" ##"

To summarize - 3 options.
1) Default Studio URL:
<a href="##Unsubscribe RedirectURL="http://studio.m.mdl.io/unsubscribe" ##">Unsubscribe</a>

2) Full URL Provided (used mostly for custom unsubs:
<a href="http://< full URL from settings">Unsubscribe</a>

3) Use campaign domain and page inside of campaign:
<a href="##Unsubscribe RedirectURL="http://dvnnnnn.m.mdl.io/<campaignID>/ 
< pagename>.html"##">Unsubscribe</a>

**Comment by daver78** [comment]
@paagla123 Removing me from assignees so my discrete tasks are clear.  Add me back if you need me to do something.

**Comment by coa0329** [comment]
@paagla123 

Fixed on stage.

I set the value of defUnsubscribe variable to be 
##Unsubscribe RedirectURL="http://studio.m.mdl.io/unsubscribe"## 
in all accounts.

**Comment by paagla123** [comment]
Works now - thanks
